### PR TITLE
Add slack link for users

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
Should we rename the slack server BTW?

### CODE REVIEW

**Review by enjoy-binbin — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -9,6 +9,9 @@ contact_links:
   - name: Chat with us on Matrix?
     url: https://matrix.to/#/#valkey:matrix.org
     about: We are on Matrix too!
+  - name: Chat with us on Slack?
+    url: https://valkey-oss-developer.slack.com/archives/C071Q2B3RNC
```
Comment (madolson) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    url: https://join.slack.com/t/valkey-oss-developer/shared_invite/zt-2nxs51chx-EB9hu9Qdch3GMfRcztTSkQ
```
I think you have to use this link? If you aren't already on the slack you will just get an error.

Comment (Yury-Fridlyand) [reply .github/ISSUE_TEMPLATE/config.yml]: Thanks!

**Comment by madolson** [comment]
> Should we rename the slack server BTW?

We have chosen to not change the domain historically because it changes the URL, but I updated the name of the slack server to `Valkey community`.

**Comment by zuiderkwast** [comment]
Does anyone use Discord and Matrix?

It can be hard for users to find us if they have to join all of them before they know which chat is active. Maybe we should delete the other links?

**Comment by madolson** [comment]
> Does anyone use Discord and Matrix?

People use matrix, and as of right now I know glide folks like to use discord. So AFAIK all three are used.

**Review by madolson — APPROVED** [review]

**Pull request merged**